src/main/java/qna/domain/BaseTimeEntity.java

```diff
@@ -0,0 +1,22 @@
+package qna.domain;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@EntityListeners(AuditingEntityListener.class)
+@MappedSuperclass
+public abstract class BaseTimeEntity {
```

Comment:
```suggestion
public class BaseTimeEntity {
```

중복코드 제거를 위한 audit 활용 좋습니다!

단독 활용 방지를 위해 abstract 를 사용하신 것이라고 생각하는데요,
`@MappedSuperclass` 가 달려 있어서 괜찮지 않을까요?

개인적으로는 abstract method를 만들어서 용도에 맞게 사용하는게 더 좋다고 생각해요.

Reply:
저도 동휘님 의견에 완전히 공감하며  `abstract를 제거` 하였는데요! 

제가 아직 JPA를 활용 한 협업에 대한 경험이 없다 보니 궁금한 부분이 
생겨 질문을 남기게 되었습니다.

```
[질문]
JPA에 대한 이해가 부족한 개발자와 같이 업무를 한다고 했을 때 해당 클래스를 
인스턴스로 생성하지 않도록 제한하는 것에 추천 할 만한 방법이 있을까요?
```

동휘님의 소중한 의견 부탁드립니다. 감사합니다!

Reply:
윤민님의 말씀대로 그러한 스펙을 부여하려면 abstract를 활용하는게 좋을 수 있겠네요.
사실 제 경험상 `BaseTimeEntity` 처럼 audit을 담당하는 클래스의 인스턴스를 생성해서 쓴 걸 본 적은 없어요.
아니면 생성자를 죄다 private로 바꾸면 생성 자체는 안되겠는데 audit에서 그렇게 활용해본 적이 없으니 정상 작동 여부를 말씀드리긴 어렵겠네요.

질문해주신 내용처럼 그러한 스펙을 강제해야겠다는 생각이 더 강하다면 abstract 를 사용하는게 더 좋을 수 있구요.
abstract 자체의 스펙을 지키지 않는게 불편하면 저처럼 제거를 하는게 맞는 것 같아요.

저희 팀 시니어 분이 항상 하시는 말씀이 `정답은 없어도 오답은 있다` 인데요,
이런 경우는 무엇을 하더라도 오답은 아니니까 더 적절한걸 스스로 판단해보시면 되겠습니다.

Reply:
경험까지 함께 공유해주신 소중한 의견 잘 메모하고 참고하겠습니다.
정말 감사드립니다! ㅠㅠ 🙇‍♂️